### PR TITLE
Updated readme github buttons

### PR DESCRIPTION
**Pull request by swathiswaminathan**
Added a 'open in cloudshell' feature into the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,22 @@ gke-on-prem-cluster-f1-pvc-bcc115d6-6472-11e8-a9b6-42010a800140  us-west1  regio
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```
Comment (emailbob) [inline README.md]: `[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

Comment (swathiswaminathan) [reply README.md]: please test this

**Comment by maxwelllincoln** [comment]
build this please

**Comment by chrislovecnm** [comment]
Why is CI falling?

**Comment by maxwelllincoln** [comment]
test this please

**Comment by chrislovecnm** [comment]
@swathiswaminathan please rebase

**Comment by swathiswaminathan** [comment]
Rebased. please test this.

**Comment by chrislovecnm** [comment]
@maxwelllincoln you need to make sure the person does a rebase and not a merge. Please and thank you

**Comment by chrislovecnm** [comment]
You get stuff like this

https://github.com/GoogleCloudPlatform/gke-enterprise-demo/commit/511981144d3aa4a5bcc44a552f0b69d92d79847c

**Pull request merged**